src/screen/addressScreen/editAddress.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {View, Text, TextInput, StyleSheet, ScrollView} from 'react-native';
+import {Button, Header} from 'components';
+import {Colors} from 'theme';
+
+const EditAddress = ({navigation, route}) => {
+  const item = route.params;
```

Comment:
instead of using **item.item.propertyName**, 
update main variable here, so that you can directly access propertyName from one level of nesting

```
eg. 
const address = route.params?.item?.item;
<TextInput style={styles.informationInput} value={address.Fullname} />
```

and again, variable/property names should be camelCase

Reply:
made Fullname --> fullName.
updated route.params?.item